pontoon/base/templates/landing.html

```diff
@@ -16,7 +16,7 @@
         <li><a href="https://developer.mozilla.org/en-US/docs/Localizing_with_Pontoon" target="_blank">Help</a></li>
         <li id="admin"{% if not perms.base.can_manage %} class="hidden"{% endif %}><a href="{{ url('pontoon.admin') }}">Admin</a></li>
         {% if not user.is_authenticated() %}
-        <li id="sign-in"><a href="">Sign in</a></li>
+        <li id="sign-in"><a href="{{ provider_login_url(request) }}">Sign in</a></li>
```

Comment:
Shouldn't this actually be `{{ url('pontoon.persona-migration') }}`? Otherwise you can only sign in using FxA.